crawl-ref/source/state.h

```diff
@@ -30,6 +30,16 @@ struct god_act_state
     int      depth;
 };
 
+struct map_mode_info_t
+{
+public:
```

Comment:
Leave out the `public` here, since it's the default. I see there's another struct in this file with an explicit `public`, but it's not the norm.